skins/macaronswap.scss

```
@@ -0,0 +1,9 @@
+.pancake {
```

The css selector should similar to the skin file name. `skin.scss` `.skin {}`.
```suggestion
.macaronswap {
```